libbeat/outputs/discard/docs/discard.asciidoc

```diff
@@ -0,0 +1,34 @@
+[[discard-output]]
+=== Configure the Discard output
+
+++++
+<titleabbrev>Discard</titleabbrev>
+++++
+
+The Discard output throws away data.
+
+WARNING: The Discard output should be used only for development or
+debugging issues.  Data is lost.
+
+This can be useful if you want to work on your input configuration
+without needing to configure an output.  It can also be useful to test
+how changes in input and processor configuration affect performance.
```

Comment:
[Doc lint]
```suggestion
This can be useful if you want to work on your input configuration
without needing to configure an output. It can also be useful to test
how changes in input and processor configuration affect performance.
```

This happens in some other places as well, but I don't want to flood the PR with whitespace comments/suggestions.